infra/docker-compose/dev/docker-compose.yml

```diff
@@ -83,8 +83,8 @@ services:
         cp /.kube/kubeconfig.yaml /.kube/config
         sed -i 's/127.0.0.1/server/' /.kube/config
         kubectl cluster-info
-        curl -sL https://istio.io/downloadIstioctl | ISTIO_VERSION=1.12.4 sh -
-        /.istioctl/bin/istioctl install -y -f /minimal-operator.yaml
```

Comment:
Why has this config file application been removed?

Reply:
I'll remove `minimal-operator.yaml` file. I don't see this required anymore in the Istio documentation.